pkg/apis/ibmcloud/v1alpha1/ibmcloudmachineproviderstatus_types.go

```diff
@@ -1,62 +0,0 @@
-/*
```

Comment:
why this file is being deleted? I think we still need this?

Reply:
openstack provider doesn't have this 

https://github.com/kubernetes-sigs/cluster-api-provider-openstack/tree/master/config/crds
I thought maybe we don't need either

Reply:
Can we keep this for now, as this file was auto-generated by https://kubernetes-sigs.github.io/cluster-api/provider_implementations/generate_crds.html

Reply:
yes, I know seems the kubebuilder we used have some issue

The yaml actually generated from the pkg/apis/ibmcloud/v1alpha1/ibmcloudmachineprovider*_types.go and we need adjust later

anyway, I restore the files